RELEASE_PROCESS.md

```diff
@@ -38,16 +38,10 @@ Then you can copy and paste the commands below without worrying about mistyping
     ```
 
 4. Find the tag you just pushed [on GitHub](https://github.com/allenai/allennlp/tags) and
-click edit. Now copy over the latest section from the `CHANGELOG.md`. And finally, add a section called "Commits" with the output from this:
+click edit. Now copy over the latest section from the `CHANGELOG.md`. And finally, add a section called "Commits" with the output a command like the following:
 
     ```bash
-    git log `git describe --always --tags --abbrev=0 HEAD^^`..HEAD^ --oneline
-    ```
-
-    Or, if you're using fish,
-
-    ```fish
-    git log (git describe --always --tags --abbrev=0 HEAD^^)..HEAD^ --oneline
```

Comment:
I like having this because I'm a fisher, so I can just copy and paste it. That said, I'm not opposed to removing it. I could always just stuff this into a function in my dotfiles.

Reply:
I am too.  I only deleted it because my change made the command identical for fish and bash.

Reply:
Pete--I added a fish command back in.  I recommend taking a second look.